### DISCUSSION TITLE
FRM Open Source

### DISCUSSION THREAD

**Discussion by euluizaraujo** [juspay/hyperswitch#3062] [Q&A] [answered]
Hi, I noticed a change in the changelog regarding FRM functionality/integration, is the FRM integration/functionality now available in the self-hosted (opensource) version?

**Comment by SanchithHegde** [comment, marked as answer]
Hey @euluizaraujo, yes we had the FRM integration merged to the `main` branch only recently. It is indeed available in the self-hosted version, we will be testing it out on our test environments and ironing out bugs (if any) over the next few days.

**Reply by euluizaraujo** [reply]
@SanchithHegde perfect, so the FRM integration is already open source but pending review, right?  Should this happen this month?

**Reply by SanchithHegde** [reply]
Yeah, I'd expect it to be done in a month's time.
